src/Transports/MassTransit.AmazonSqsTransport/Configuration/Configuration/AmazonSqsHostConfiguration.cs

```diff
@@ -42,6 +42,7 @@ public class AmazonSqsHostConfiguration :
             ReceiveTransportRetryPolicy = Retry.CreatePolicy(x =>
             {
                 x.Handle<AmazonSqsTransportException>();
+                x.Handle<AmazonSqsConnectionException>();
```

Comment:
this should be enough to start retrying

Reply:
As per my previous comment, if the exception is handled and never re-thrown in the consumer loop, when would this retry anything?

Reply:
also, you'll need to specify if exception is transient:
like:
https://github.com/MassTransit/MassTransit/blob/9e21c29bba03ddb96d6eb17117c79e8691a85a43/src/Transports/MassTransit.KafkaIntegration/Exceptions/KafkaConnectionException.cs#L31
in:
https://github.com/MassTransit/MassTransit/blob/9e21c29bba03ddb96d6eb17117c79e8691a85a43/src/Transports/MassTransit.KafkaIntegration/Exceptions/KafkaConnectionException.cs#L31

and this should be enough to retry

Reply:
@NooNameR the transport already raises all AmazonSqsConnectException as transient, although implicitly by using the overloaded ctors.

Reply:
yeah, you are right, my bad